tests/test_e2e_ocr_conversion.py

```diff
@@ -97,11 +101,68 @@ def test_e2e_conversions():
         engines.append((OcrMacOptions(), True))
         engines.append((OcrMacOptions(force_full_page_ocr=True), True))
 
+    # DeepSeek OCR - requires GPU (CUDA or MPS) and transformers
+    # Only run if explicitly enabled via environment variable
+    # Set DOCLING_TEST_DEEPSEECOCR=true to include DeepSeek-OCR tests
```

Comment:
instead of the ENV, could we detect if deepseek-ocr is runnable?

Reply:
What do you mean by runnable?